.circleci/config.yml

```diff
@@ -235,6 +235,14 @@ jobs:
     executor: node-executor
     steps:
       - checkout
+      - run:
+          name: Check if yarn.lock changed
+          command: |
+            if git diff HEAD~1 --name-only | grep -q "yarn.lock"; then
+              echo "yarn.lock changed, checking vulnerabilities"
+            else
+              circleci-agent step halt
+            fi
```

Comment:
probably an overkill but maybe at some point we could move these security steps to an orb

Reply:
Oh yah, that's a great idea. Would take two seconds really. cc @brainbicycle 

Let me know if y'all implement this over in eigen; if so thats now three repos duplicating code. No need for it to live there if we can drop it in the artsy orb.